packages/notifications/examples/basic.md

```diff
@@ -129,8 +129,8 @@ const initialState = {
   <Row>
     <Col md={4}>
       <Well isRecessed>
-        <div role="group" aria-label="alert types radio">
-          <StyledMd>Alert Types</StyledMd>
+        <div role="group" aria-label="notification types radio">
+          <StyledMd>Notification Types</StyledMd>
```

Comment:
Update this example to with a `Default` radio button (selected by default) which sets the notification type to `undefined`.